src/utils/bundleFeedUtils.ts

```diff
@@ -100,4 +100,11 @@ export namespace bundleFeedUtils {
 
         return feedUtils.getJsonFeed(context, url);
     }
+
+    export function overwriteExtensionBundleVersion(hostJson: IHostJsonV2, expectedRange: string, newRange: string): void {
+        if (hostJson.extensionBundle) {
+            if (hostJson.extensionBundle.version === expectedRange)
```

Comment:
Can we use braces here? Or change the condition in the outer `if` to `hostJson.extensionBundle?.version === expectedRange` and remove the inner `if`? Non-block-style `if`s too often lead to hard-to-spot bugs...